content/scalability/docs/control-plane.md

```diff
@@ -0,0 +1,75 @@
+# Kubernetes Control Plane
+
+The Kubernetes control plane is the Kubernetes components such as the Kubernetes API Server, Kubernetes Controller Manager, and other components that are required for Kubernetes to funciton. Scalability limits of these components are different depending on what you’re running in the cluster, but the areas with the biggest impact to scale include the Kubernetes version, utilization, and individual Node scaling.
```

Comment:
1st sentence, "The Kubernetes control plane consists of...", replace "funciton" with "function". but the areas "that have the biggest impact on scaling" include...